tests/functional/cylc-review/00-basic.t

```diff
@@ -202,12 +191,12 @@ cylc_ws_json_greps "${TEST_NAME}.stdout" "${TEST_NAME}.stdout" \
 
 #-------------------------------------------------------------------------------
 # Data transfer output check for a suite run directory with only a "log/db"
-COPY_NAME="${SUITE_NAME}-copy"
-cylc register "${COPY_NAME}"
+COPY_NAME="${WORKFLOW_NAME}-copy"
 
-CYLC_RUN_DIR="$(cylc get-global-config --print-run-dir)"
-mkdir "${CYLC_RUN_DIR}/${COPY_NAME}/log/"
-cp "${SUITE_RUN_DIR}/log/db" "${CYLC_RUN_DIR}/${COPY_NAME}/log/"
+CYLC_RUN_DIR="${HOME}/cylc-run"
+mkdir -p "${CYLC_RUN_DIR}/${COPY_NAME}/log/"
+cp "${WORKFLOW_RUN_DIR}/log/db" "${CYLC_RUN_DIR}/${COPY_NAME}/log/"
+cp "${WORKFLOW_RUN_DIR}/flow.cylc" "${CYLC_RUN_DIR}/${COPY_NAME}/"
```

Comment:
The Cylc 8 switch is tripped by the file structure of the installed workflow containing either `flow.cylc` or `log/scheduler`.